CONTRIBUTING.md

```diff
@@ -34,4 +34,4 @@ To contribute code to this project, pleas euse the following workflow:
 ## Documentation
 
 Documentation for this charm is currently maintained as part of the Charmed Kubernetes docs.
-See [this page](https://github.com/charmed-kubernetes/kubernetes-docs/blob/master/pages/k8s/charm-etcd.md)
+See [this page](https://github.com/charmed-kubernetes/kubernetes-docs/blob/main/pages/k8s/charm-etcd.md)
```

Comment:
The etcd link here doesn't make much sense. Seems like it was a mistake?

I'm not asking you to change it, but thought I should point it out.

Reply:
easy enough.  I'll fix it.

Reply:
Actually: fixing it would mean referencing a non-inclusive name.  I'll add an bug instead

Reply:
actually i decided to fix it in another PR which gets merged before this one
https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/210